docs/scripts/content/styled-system.ts

```diff
@@ -127,14 +132,17 @@ const getJSDocs = (node: TypeAliasDeclaration) => (sourceFile: SourceFile) => {
 
     if (!hasJSDoc(member)) return
 
-    member.jsDoc.forEach(({ tags }) =>
+    member.jsDoc.forEach(({ tags, comment }) => {
+      data.description =
+        typeof comment === "string" ? comment : comment?.join("\n")
```

Comment:
Shouldn't we use `isString` from @yamada-ui/utils?

Reply:
@taroj1205 

You cannot use `@yamada-ui/utils` or `@yamada-ui/react` here.